tests/util/random_reseed.py

```diff
@@ -0,0 +1,10 @@
+import numpy
```

Comment:
@selwyn96 I think this file needs to be moved to `src/qonnx/util/random_reseed.py` and the corresponding lines in `setup.cfg` need to become:
```
[options.entry_points]
pytest_randomly.random_seeder =
    qonnx = qonnx.util.random_reseed.reseed
    ```